src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -77,6 +79,7 @@ def run_python_test(test_target, pytest, python_setup, source_root_config):
   interpreter_constraint_args = parse_interpreter_constraints(
     python_setup, python_target_adaptors=all_targets
   )
+  interpreter_search_paths = text_type(create_path_env_var(python_setup.interpreter_search_paths))
```

Comment:
Add a TODO that this won't work when executing remotely, because we'll need to do path resolution on the remote system or establish known paths.

Reply:
I don't think anything in source code is going to have to change for this. All we'll have to do is change the options value for `interpreter_search_paths`, and everything should work!

Because the TODO is just changing config values and not actual `src` code, I'm going to not add a TODO.